config/DoxygenModules.txt

```diff
@@ -0,0 +1,316 @@
+ *
+ * To add MobiledgeX Package, select the **+** icon and click on **“Add package from git URL…”** 
+ *
+ *![](https://developers.mobiledgex.com/assets/unity-sdk/add-git-url.png)
+ *
+ * Enter [https://github.com/mobiledgex/edge-cloud-sdk-unity.git](https://github.com/mobiledgex/edge-cloud-sdk-unity.git) in the text field, which will automatically start the process of importing the package into your application. 
+ * 
+ * 
+ * \section setup Setup
+ *
+ * Once you have successfully imported the Unity package, you will see a new tab as part of the Unity menu labeled **MobiledgeX**
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/mobiledgex-menu.png)
+ * 
+ * Click **Setup**, which will open a new Unity window asking you for your application's
+ * * organization name
+ * * app name
+ * * app version number 
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/mobiledgex-unity-window.png)
+ * 
+ * After you provide your application credentials, click the setup button, which will communicate with the DME to verify that your application definition exists on the MobiledgeX console. If successful, your project will be set up with the correct plugins and resources necessary to use our APIs. You can verify if these files were generated correctly by looking in the Plugins and Resources folders of your project. 
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/generated-plugins.png)
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/generated-resources.png)
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/mobiledgex-settings.png)
+ * 
+ * **Important**: Make sure your Resources/MobiledgeXSettings.asset file has the correct information for your application. 
+ * 
+ * \section platform_specific Platform Specific
+ *
+ * ### Android
+ * 
+ * The minimum API we support for Android is API Version 24. In your player settings, make sure to set the minimum API to 24, otherwise you will be unable to build your project. 
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/android_version_error.png)
+ * 
+ * \section known_issues Known Issues 
+ * 
+ * If you recieve the following error and cannot compile your Unity project, restart Unity.
+ * 
+ * ![](https://developers.mobiledgex.com/assets/unity-sdk/metadata_error.png)
+ * 
+ * 
+ * \page example_usage Example Usage
+ * Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package 
+ * 
+ * \section edge_url Getting Edge Connection Url
+ *
+ * 
+ * MobiledgeX SDK uses the device Location and [the device's MCC-MNC ID (if avaliable)](https://developers.mobiledgex.com/getting-started/connecting-client-app#distributed-matching-engine) to connect you to the closest Edge cloudlet where you application instance is deployed.
```

Comment:
Not sure if this is something specific for Doxygen or not, but the brackets around "the device's MCC-MNC ID (if avaliable)" look weird.